policyengine_canada/variables/household/person/childcare_expenses_claimed_by_another_person.py

```diff
@@ -0,0 +1,9 @@
+from policyengine_canada.model_api import *
+
+
+class childcare_expenses_claimed_by_another_person(Variable):
+    value_type = float
+    entity = Person
+    label = "Amount of Childcare received which was claimed by another person"
+    unit = CAD
+    definition_period = YEAR
```

Comment:
from policyengine_canada.model_api import *


class childcare_received(Variable):
    value_type = float
    entity = Person
    label = "Amount of Childcare received which was claimed by another person"
    unit = CAD
    definition_period = YEAR